### PR TITLE
Docs updates for Croeseid-v0.9.1-upgrade 

### PR DESCRIPTION
**Pull request by lezzokafka**
Updated `getting-started/croeseid-testnet.md` to v0.9.1-croeseid along with some links redirections.

### CODE REVIEW

**Review by tomtau — COMMENTED** [review]
aren't the current instructions -- just to download 0.9.1 -- for state sync only?
if one wanted to run the full sync, they'll need to get both 0.8.1 and 0.9.1 ?

**Comment by lezzokafka** [comment]
> aren't the current instructions -- just to download 0.9.1 -- for state sync only?
> if one wanted to run the full sync, they'll need to get both 0.8.1 and 0.9.1 ?

Sure. Let me also add remarks for running a full node.

File: docs/getting-started/croeseid-testnet.md [hunk 1]
```diff
@@ -112,19 +114,23 @@ Before kick-starting your node, we will have to configure your node so that it c
...
 Follow the below optional steps to enable state-sync.
-:::
 
-- (Optional) For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
-  Also, we suggest using `persistent_peers` instead of `seeds` to to provide stable state-sync experience.
+
+- For state-sync configuration, in `~/.chain-maind/config/config.toml`, please modify the configurations of [statesync] `enable`, `rpc_servers`, `trust_height` and `trust_hash` by:
+
   ```bash
   $ LASTEST_HEIGHT=$(curl -s https://testnet-croeseid.crypto.org:26657/block | jq -r .result.block.header.height); \
```
Comment (CeruleanAtMonaco) [inline docs/getting-started/croeseid-testnet.md:135]: LASTEST_HEIGHT <- typo?

Comment (lezzokafka) [reply docs/getting-started/croeseid-testnet.md:135]: Its all good 👍

**Review by tomtau — APPROVED** [review]

**Pull request merged**